### ISSUE TITLE
Model accuracy issue

### ISSUE THREAD

**Issue by ingjieye** [look4pritam/tfmtcnn#15]
Hi. I've trained mtcnn model with your code. The detection accuracy on FDDBDataset is 94.29%. 

Then I tested three models you provided under `tfmtcnn/models/mtcnn` with FDDBDataset. 

The `accurate` one got 97.04% detection accuracy, the `fast` one  got 96.06% accuracy.

Any idea how to improve the detection accuracy? How did you train these three model? 

Btw I've replace `base_number_of_attempts` from 5000 to 100 as discussed on #12 , will it affect the accuracy of the model?

Thanks.

**Comment by look4pritam** [comment]
**First point**
Reducing base_number_of_attempts from 5000 to 100 results in less data augmentation and hence accuracy decreases. 

**Second point**
MTCNN models are having just 4 layers, hence models get over fitted easily. To avoid this, I have calculated accuracy ON FDDB dataset after some predefined epoch. Model with better accuracy is selected for next model training.

This two things will improve your accuracy also.

**Comment by ingjieye** [comment]
Sorry, I'm very new to deep learning and Tensorflow. As you said 

> I have calculated accuracy ON FDDB dataset after some predefined epoch. Model with better accuracy is selected for next model training.

Is this done automatically or if not how do I `selected` model with better accuracy? 
Thank you!

**Comment by look4pritam** [comment]
It is done manually. Model is trained and then evaluated on fddb. If accuracy is increasing then train again and evaluate. It avoids model overfitting.
